composer.json

```diff
@@ -57,7 +57,8 @@
         "symfony/serializer": "^3.0|^4.0",
         "symfony/yaml": "^3.0|^4.0",
         "webmozart/assert": "^1.0",
-        "webmozart/path-util": "^2.3"
+        "webmozart/path-util": "^2.3",
+        "alibabacloud/wafopenapi": "^1.7"
```

Comment:
any reason for using that package (almost not downloaded, not starred, ...) instead of `alibabacloud/client`?

Reply:
- alibabacloud/client is a psr7 lib like guzzle. Doesn't contains the feature of his any product request parameters.
- there's a package named alibaba/sdk, it contains all alibaba's hundreds of products. seems too heavy for us.